packages/optimizely-sdk/lib/core/audience_evaluator/index.tests.js

```diff
@@ -323,12 +332,248 @@ describe('lib/core/audience_evaluator', function() {
 
         it('should not be able to overwrite built in `custom_attribute` evaluator', function() {
           assert.isTrue(
-            audienceEvaluator.evaluate(['0'], audiencesById, {
+            audienceEvaluator.evaluate(['0'], audiencesById, getMockUserContext({
               browser_type: 'chrome',
-            })
+            }))
           );
         });
       });
     });
+
+    context('with odp segment evaluator', function() {
+      describe('Single ODP Audience', () => {
+        const singleAudience = {
+          "conditions": [
+            "and",
+            {
+                "value": "odp-segment-1",
+                "type": "third_party_dimension",
+                "name": "odp.audiences",
+                "match": "qualified"
+            }
+          ]
+        };
+        const audiencesById = {
+          0: singleAudience,
+        }
+        const audience = new AudienceEvaluator();
+        
+        it('shoud evaluate to true if segment is found', () => {
```

Comment:
```suggestion
        it('should evaluate to true if segment is found', () => {
```
fix same for others below